lucet-wiggle/src/lib.rs

```diff
@@ -40,13 +40,22 @@ pub mod runtime {
         fn has_outstanding_borrows(&self) -> bool {
             self.bc.has_outstanding_borrows()
         }
-        fn is_borrowed(&self, r: Region) -> bool {
+        fn is_shared_borrowed(&self, r: Region) -> bool {
```

Comment:
this isnt quite correct - in addition to the GuestMemory api changing, the underlying borrow checker has evolved as well, and because it was copy-pasted in a variety of places it now comes from the `wiggle-borrow` crate. You can delete `lucet-wiggle/src/borrow.rs` and wire up `wiggle_borrow::BorrowChecker` here instead please

Reply:
Resolved by awortman in #614; thanks!